Assets/HotmaxEditorVR/handColliderFollow.cs

```diff
@@ -35,7 +35,6 @@ void updateSelectedObjectIsActive(bool value)
     void Update()
     {
         transform.position = selectorHand.position;
-        transform.rotation = selectorHand.rotation;
```

Comment:
whats this fixing?

Reply:
what did you need that line for before?

Reply:
I didn't need it for anything. it won't be missed